package-lock.json

```diff
@@ -1,435 +1,5 @@
 {
   "name": "@luckbox/eslint-rules",
-  "version": "2.2.5",
-  "lockfileVersion": 1,
-  "requires": true,
```

Comment:
This looks strange - did `npm install` remove all these lines?

Reply:
yes since the we now have peerDependencies instead of just dependencies